### PR TITLE
chore: Changes in action execution analytics event

### PR DESCRIPTION
**Pull request by vishnu-gp**
## Description

This PR introduces some changes in action execution analytics event to,
- Not query details from DB or log the event in CE if telemetry is disabled.
- Have an option to log event data only while not firing Analytics in EE even if telemetry is disabled.

## Type of change

- New feature (non-breaking change which adds functionality)

## How Has This Been Tested?

- Locally

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: app/server/appsmith-server/src/main/java/com/appsmith/server/services/ce/NewActionServiceCEImpl.java [hunk 1]
```diff
@@ -1017,6 +1017,14 @@ private ActionExecutionResult addDataTypesAndSetSuggestedWidget(ActionExecutionR
         return result;
     }
 
+    /**
+     * Overridable method to handle system events in Enterprise Edition
```
Comment (mohanarpit) [inline app/server/appsmith-server/src/main/java/com/appsmith/server/services/ce/NewActionServiceCEImpl.java:1021]: Please remove this comment :)

Comment (vishnu-gp) [reply app/server/appsmith-server/src/main/java/com/appsmith/server/services/ce/NewActionServiceCEImpl.java]: Done

**Review by trishaanand — APPROVED** [review]

**Pull request merged**